### ISSUE TITLE
EventGridTrigger not supported?

### ISSUE THREAD

**Issue by jubueche** [microsoft/vscode-azurefunctions#1747]
Hi,
I tried to create a trigger function responding to EventGrid events, but it seems there is not a template for it in the current VS code extension. I am using Ubuntu 18.10 and downloaded the extension roughly two weeks ago.
[This](https://docs.microsoft.com/en-us/azure/azure-functions/functions-debug-event-grid-trigger-local) indicates that standard Visual Studio has this functionality. So is this only possible on Windows?

**Comment by EricJizbaMSFT** [comment]
We had a few issues for this already, not entirely sure why they were closed: https://github.com/microsoft/vscode-azurefunctions/issues/792 and https://github.com/Azure/azure-functions-templates/issues/803

Filed a new issue on the templates repo: https://github.com/Azure/azure-functions-templates/issues/915

**Comment by jubueche** [comment]
Is this supported in standard Visual Studio?

**Comment by jubueche** [comment]
Any news on this? We would like to start working on our EventGrid triggered function not coding from Azure portal.

**Comment by EricJizbaMSFT** [comment]
Currently waiting on the upstream issue
